### ISSUE TITLE
Error running the module with jhipster 4.13.1

### ISSUE THREAD

**Issue by poolebu** [sudheerj/generator-jhipster-primeng#31]
Hello! I wanted to try this module, I have tried several combinations of components in the generator, but always getting this error
```
Error: StaticInjectorError[NgZone]: 
  StaticInjectorError[NgZone]: 
    NullInjectorError: No provider for NgZone!
    at _NullInjector.get (webpack-internal:///./node_modules/@angular/core/esm5/core.js:1190)
    at resolveToken (webpack-internal:///./node_modules/@angular/core/esm5/core.js:1478)
    at tryResolveToken (webpack-internal:///./node_modules/@angular/core/esm5/core.js:1420)
    at StaticInjector.get (webpack-internal:///./node_modules/@angular/core/esm5/core.js:1291)
    at resolveToken (webpack-internal:///./node_modules/@angular/core/esm5/core.js:1478)
    at tryResolveToken (webpack-internal:///./node_modules/@angular/core/esm5/core.js:1420)
    at StaticInjector.get (webpack-internal:///./node_modules/@angular/core/esm5/core.js:1291)
    at resolveNgModuleDep (webpack-internal:///./node_modules/@angular/core/esm5/core.js:11080)
    at _createClass (webpack-internal:///./node_modules/@angular/core/esm5/core.js:11125)
    at _createProviderInstance$1 (webpack-internal:///./node_modules/@angular/core/esm5/core.js:11091)
(anonymous) @ webpack-internal:///./src/main/webapp/app/app.main.ts:12

```
This is my `.yo-rc.json `
```
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "io.catwizard.autdit.test1"
    },
    "jhipsterVersion": "4.13.1",
    "baseName": "jhipster",
    "packageName": "io.catwizard.autdit.test1",
    "packageFolder": "io/catwizard/autdit/test1",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "ehcache",
    "enableHibernateCache": false,
    "clusteredHttpSession": false,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "h2Memory",
    "prodDatabaseType": "mysql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSocialSignIn": false,
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "XXX",
    "clientFramework": "angularX",
    "useSass": false,
    "clientPackageManager": "yarn",
    "applicationType": "monolith",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "enableTranslation": false
  }
}
```

**Comment by GuiF007** [comment]
See comments of bug #26 !

**Comment by sudheerj** [comment]
Issue is fixed in 2.0.19 release. Due to duplicate Angular dependencies then it is not able to resolve the providers. Thanks for reporting.

**Comment by poolebu** [comment]
Tested with `+ generator-jhipster-primeng@2.0.22` and it worked. Thank you!
